test/python/utils.py

```diff
@@ -11,3 +21,52 @@ def concatenate(*args):
     for arg in args:
         result += (bytes([len(arg)]) + arg)
     return result
+
+def validate_displayed_message(client: SpeculosBackend, num_screen_skip: int):
+    for _ in range(num_screen_skip):
+        client.right_click()
+    client.both_click()
+
+# DERIVATION PATHS CALCULATIONS
+
+class BtcDerivationPathFormat(IntEnum):
+    LEGACY      = 0x00
+    P2SH        = 0x01
+    BECH32      = 0x02
+    CASHADDR    = 0x03 # Deprecated
+    BECH32M     = 0x04
+
+def pack_derivation_path(derivation_path: str) -> bytes:
+    split = derivation_path.split("/")
+    assert split.pop(0) == "m", "master expected"
+    derivation_path: bytes = (len(split)).to_bytes(1, byteorder='big')
+    for i in split:
+        if (i[-1] == "'"):
+            derivation_path += (int(i[:-1]) | HARDENED_INDEX).to_bytes(4, byteorder='big')
+        else:
+            derivation_path += int(i).to_bytes(4, byteorder='big')
+    return derivation_path
+
+def bitcoin_pack_derivation_path(format: BtcDerivationPathFormat, derivation_path: str) -> bytes:
+    assert isinstance(format, BtcDerivationPathFormat)
+    return format.to_bytes(1, "big") + pack_derivation_path(derivation_path)
+
+
+# CURRENCY CONFIG CALCULATIONS
+
+def create_sub_currency_config(ticker: str, number: int) -> bytes:
+    sub_coin_config: buffer = b""
+    sub_coin_config += (len(ticker)).to_bytes(1, byteorder='big')
+    sub_coin_config += ticker.encode('utf-8')
+    sub_coin_config += (number).to_bytes(1, byteorder='big')
+    return sub_coin_config
+
+def create_currency_config(ticker: str, application_name: str, sub_coin_config: bytes = b"") -> bytes:
+    coin_config: buffer = b""
+    coin_config += (len(ticker)).to_bytes(1, byteorder='big')
```

Comment:
This could be false. `len(string) != len(string.encode())`.

As these lines are kinda repetitive, I suggest:
```python
for part in [ticker.encode(), application_name.encode(), sub_coin_config]:  # "utf-8" is the default encoding in Python3
    coin_config += len(part).to_bytes(1, byteorder="big") + part
```

Reply:
Nice catch